src/map.h

```diff
@@ -1039,7 +1040,11 @@ class map
         // Traps: 3D
         void trap_set( const tripoint &p, const trap_id &type );
 
+        void partial_con_set( const tripoint &p, const partial_con &con );
```

Comment:
Minor, but maybe don't put these declarations right in the middle of the trap declarations?